### PR TITLE
feat(dsl): added the reduced class set

### PR DESCRIPTION
**Pull request by programminx-askui**

### CODE REVIEW

**Review by mlikasam-askui — CHANGES_REQUESTED** [review]
we need to increase the api version and deprecate the old versions because of the dsl  breaking changes

File: docs/docs/api/03-Filters/withtext.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ Filters for similar (doesn't need to be a 100% equal) text.
 'texst' === withText('text') => true
 'texts' === withText('text') => true
 
-// usually false'atebxtc' === withText('text') => false
+// usually false
```
Comment (JohannesDienst-askui) [inline docs/docs/api/03-Filters/withtext.md:18]: adi-wan asked me to remove this comment, so I did.

File: docs/docs/api/01-API/table-of-contents.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 ---
 displayed_sidebar: apiSidebar
+custom_edit_url: null
```
Comment (JohannesDienst-askui) [inline docs/docs/api/01-API/table-of-contents.md:3]: Please remove this as it breaks the `Edit this page` button at the bottom of the page.

**Review by JohannesDienst-askui — CHANGES_REQUESTED** [review]

**Comment by JohannesDienst-askui** [comment]
Except of the two subtle things I do approve 👍

File: docs/docs/api/03-Filters/icon.md [hunk 3]
```diff
@@ -12,5 +12,5 @@ You can combine it with the 'withText' command to look for a specific icon.
 icon().withText('plus')
 ```
 
-**Note:** This is an alpha feature. The prediction of the icon name is sometimes unstable. Use custom elements as an alternative.
+Note: This is an alpha feature. The prediction of the icon name is sometimes unstable. Use custom elements as an alternative.
```
Comment (mlikasam-askui) [inline docs/docs/api/03-Filters/icon.md:15]: is there a reason for removing this ?

Comment (programminx-askui) [reply docs/docs/api/03-Filters/icon.md:15]: Generated.

**Comment by mlikasam-askui** [comment]
> we need to increase the api version and deprecate the old versions because of the dsl breaking changes and the api doc still need to be updated

no need since the response will be "Could not parse instruction. If you are using an askui product, "
                    "please upgrade to the newest version"

**Review by mlikasam-askui — APPROVED** [review]

**Pull request closed without merging**